.github/workflows/e2e_lighthouse_performance_acceptance_tests.yml

```diff
@@ -125,64 +75,29 @@ jobs:
             requirements_dev.in
             requirements_dev.txt
       - uses: ./.github/actions/merge
-      - name: Cache node modules
-        uses: actions/cache@v3
-        env:
-          cache-name: cache-node-modules
-        with:
-          path: /home/runner/work/oppia/yarn_cache
-          key: ${{ runner.os }}-build-${{ env.cache-name }}-${{ hashFiles('yarn.lock') }}
-          restore-keys: |
-            ${{ runner.os }}-build-${{ env.cache-name }}-
-            ${{ runner.os }}-build-
-            ${{ runner.os }}-
-      - name: Attempt to download build files artifact
-        id: download_artifact_1
-        uses: actions/download-artifact@v3
-        continue-on-error: true
-        with:
-          name: build_files
-          path: /home/runner/work/oppia/
-      - name: Unzip build files
-        if: steps.download_artifact_1.outcome != 'failure'
+      # Installing python dependencies becuase we are executing a python script for installing chrome in CI.
+      - name: Install Python deps
         run: |
-          ls -la
-          unzip build_files.zip
-          rm build_files.zip
-          echo "Files in ./:"
-          ls -la .
-          echo "Files in oppia_tools:"
-          ls -la oppia_tools
-          echo "Files in oppia:"
-          ls -la oppia
-          echo "Files in build:"
-          ls -la oppia/build
-          echo "Files in third_party:"
-          ls -la oppia/third_party
-        working-directory: /home/runner/work/oppia
-        shell: bash
-      - uses: ./.github/actions/install-oppia-dependencies
+          pip install --require-hashes --no-deps -r requirements.txt
+          pip install --require-hashes --no-deps -r requirements_dev.txt
```

Comment:
working on adding a bash script, and checking if it works fine. will update the thread here

Reply:
@DubeySandeep can you please tell me if this third-party action will work? -- https://github.com/marketplace/actions/setup-chromedriver#for-ubuntu-latest-macos-latest
if yes, please let me know which version of chromedriver works with the e2e tests and is compactible with webdriver because currently e2e tests are not functional on the latest function and developers cant run e2e tests locally on their system if they are on the latest version

Reply:
we use chromedrive 102. also, can you please confirm that we can use such third-party actions and there wouldnt be any licensing issue?

yes we are unable to run e2e tests locally (irrespective of the docker setip)

Reply:
@Shivkant-Chauhan I thought this was fixed. Is this still an issue -- can the e2e tests run locally on your machine?

Also I don't think the original point of this thread was addressed. Why do we still need to install all the deps twice just to install Chrome?